docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -188,17 +193,20 @@ used to verify the serving certificate of the apiserver.
 Finally, the default namespace to be used for namespaced API operations is placed in a file
 at `/var/run/secrets/kubernetes.io/serviceaccount/namespace` in each container.
 
-From within a pod the recommended ways to connect to API are:
+From within a Pod, the recommended ways to connect to the Kubernetes API are:
+
+  - use one of the official [client libraries](/docs/reference/client-libraries/)
+    as they handle API host discovery and authentication out of the box.
+    For Go client, the `rest.InClusterConfig()` function assists with this.
+    See [an example here](https://git.k8s.io/client-go/examples/in-cluster-client-configuration/main.go).
 
-  - run a kubectl proxy as one of the containers in the pod, or as a background
-    process within a container.  This proxies the
-    Kubernetes API to the localhost interface of the pod, so that other processes
-    in any container of the pod can access it.  See this [example of using kubectl proxy
-    in a pod](https://github.com/kubernetes/examples/tree/{{page.githubbranch}}/staging/kubectl-container/).
-  - use the Go client library, and create a client using the `rest.InClusterConfig()` and `kubernetes.NewForConfig()` functions.
-    They handle locating and authenticating to the apiserver. [example](https://git.k8s.io/client-go/examples/in-cluster-client-configuration/main.go)
+  - if you would like to query the API without an official client library, you can run `kubectl proxy`
+    as one of the containers in the pod (i.e. sidecar container), or as a background process within a container.
```

Comment:
>as one

in one

Reply:
or:

```
 as the [command](/docs/tasks/inject-data-application/define-command-argument-container/) for a container
``

Reply:
actually rephrasing this to remove "background process" recommendation and just saying sidecar proxy with `kubectl proxy` as the command.